frontend/packages/dev-console/integration-tests/features/topology/topology-smoke_test.feature

```diff
@@ -0,0 +1,218 @@
+
+@regression, @smoke, @manual
+Scenario: Create visual connection between two nodes using drag and drop
+   Given user is in the topology with atleast two nodes
+   When user scrolls over a node to see the arrow
+   And user click on the front of arrow and drag it on to the other node and drop it
+   Then user can see the arrow connecting them with head pointing to the node where the arrow is dropped
+
+@regression, @smoke
+Scenario: Context menu of nodes
+   Given user is in the topology
+   And topology has workloads
+   When user right clicks on the node 
+   And user sees context menu
+   Then user checks options
+
+@regression, @smoke
+Scenario: Zoom In in topology
+   Given user is in topology
+   And topology has workloads
+   Given user is in topology with workloads 
+   When user clicks on Zoom In option
+   Then user sees the chart area is zoomed
+
+@regression, @smoke
+Scenario: Zoom Out in topology
+   Given user is in topology
+   And topology has workloads
+   Given user is in topology with workloads 
+   When user clicks on Zoom Out option
+   Then user sees the chart area is zoomed out
+
+@regression, @smoke
+Scenario: Fit to Screen in topology
+   Given user is in topology
+   And topology has workloads
+   When user clicks on Zoom In option
+   And user sees the chart area is zoomed
+   And user clicks on Fit to Screen option
+   And user sees the nodes fitting within chart area
+
+@regression, @smoke
+Scenario: Reset view in topology
+   Given user is in topology
+   And topology has workloads
+   When user clicks on Zoom In option
+   And user sees the chart area is zoomed
+   And user clicks on Reset View option
+   Then user sees the chart area is reset to original
+
+@regression, @smoke
+Scenario: Verify Application grouping sidebar
+   Given user is in topology
+   And topology has application cluster with nodes
+   When user clicks on an applicaton grouping
+   Then user can see application sidebar
+   And user can confirm the workload information present under resources in the sidebar
+   And user can see "Add to Application" and "Delete Application" in the Action menu
+   And user right click on Application to open context menu
+   Then user can view action menu options
+
+@regression, @smoke
+Scenario: Verify Application grouping context menu
+   Given user is in topology with an application cluster
+   When user clicks on an applicaton grouping
+   And user right click on Application to open context menu
+   Then user can view "Add to Application" and "Delete Application" options
+
+@regression, @smoke
+Scenario: Display options in topology
+   Given user is in topology
+   And topology has workloads
+   When user clicks on Display Options on top of topology
+   And user sees "Pod Count", "Event Sources" if present and "Labels" under "Show" and "Expand" have options according to their presence like "Application Groupings", "Helm Releases", "Knative Services" and "Operator Groupings"
```

Comment:
These are then statements, as you are getting an output

Reply:
But we are performing actions in further `And`